apiary-metastore-listener/README.md

```diff
@@ -75,7 +91,12 @@ The following shows an example JSON message representing a "DROP_TABLE" event:
       "protocolVersion": "1.0",
       "eventType": "DROP_TABLE",
       "dbName": "some_db",
-      "tableName": "some_table"
+      "tableName": "some_table",
+      "tableLocation": "s3://table_location",
+      "tableParameters": {
+          "hive.version": "2.3.0",
```

Comment:
are these valid table parameters or made up?

Reply:
I think these are valid, we need to either amend the line saying what setup causes these to be generated or possibly only include them in one example (as in my comment for "create table" above)/

Reply:
Yeah. They are valid. Taken from: https://confluence.expedia.biz/display/HCOMDBE/BDP-1528+-+Shunting+Yard+-+Hive+Events

Reply:
Aren't those something like metastore parameters, table parameters are like:
```
      "parameters":{
         "totalSize":"3039067",
         "EXTERNAL":"TRUE",
         "numFiles":"1",
         "transient_lastDdlTime":"1551187915"
      },
```

Reply:
yes I just checked it the `event.getParameters()` and I think we're interested in `event.getTable().getParameters` so these are wrong examples atm.

Reply:
Ok. A bit tricky to choose a common pattern in that. I chose those examples coz I could select a pattern "hive.*" in them. I think the parameters can be set to anything, for this table, sure they are like above but I can also create a table with hive.* as parameters. isn't it ?

Here, we only want to demonstrate that the emitter has the ability to select parameters from the table using a common pattern. Should we leave it like this and I can update the line above to be like what Adrian suggested ?

Reply:
We discussed offline and I have changed these to my_var_one and my_var_two. Also, I updated the introduction to say what adrian suggested and removed these from other events. Only CREATE event now has tableParams and rest all events have empty parameters.